specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -1960,6 +2088,145 @@
           "format": "date-time"
         }
       }
+    },
+    "AcsAdvancedMessageChannelEventError": {
```

Comment:
We shouldn't be defining new error types, but instead use the core responseError